### DISCUSSION TITLE
How can I remap the watch and completed folder to another volume?

### DISCUSSION THREAD

**Discussion by Unrepentant-Atheist** [haugene/docker-transmission-openvpn#1996] [Q&A] [answered]
I've got a Synology NAS and I'm running this container on it. It works great. 

I got 
```
        volumes:
            - /volume1/docker/transmission/:/data
            - /volume1/docker/transmission/config:/config
```
Now I've already got a download folder on `/volume1/downloads` and a "watch" folder for when I put .torrent files into it it's loaded into the Synolog Download Station `/volume1/downloads/watch`

Is there a way to make both those folders part of Transmissio docker-compose volume, leave incomplete in /data, but complete and watch, move to `/volume1/downloads`.

**Comment by pkishino** [comment, marked as answer]
Yes, just mount those folders to where you want them.
Something like below add to current config
        volumes:
            - /volume1/downloads/watch:/data/watch
            - /volume1/downloads/completed:/data/completed
